GoogleCloudPlatform/k8s-config-connector
# Add Edgecontainer cluster/node-pool resources

### Change description


Adds resources for Edge Container API.

* EdgeContainerCluster
* EdgeContainerNodePool

### Tests you have done

* make ensure
* make ready-pr
* go test -v -tags=integration ./pkg/controller/dynamic/ -test.run TestCreateNoChangeUpdateDelete -run-tests edgecontainercluster -timeout 3600s
* node-pool integration test is not available since it need physical machine support, tested e2e manually


- [X] Run `make ready-pr` to ensure this PR is ready for review.
- [X] Perform necessary E2E testing for changed resources.
